commit b71107fb12e3c3125e0cb62c5a4f6636a80c6408
Author:     Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
AuthorDate: Tue Jun 7 11:52:37 2011 +0200
Commit:     Bjoern Michaelsen <bjoern.michaelsen@canonical.com>
CommitDate: Tue Jun 7 11:52:37 2011 +0200

    on debian-based systems, we know where our soffice binary is

Index: libreoffice-4.1.4~rc2/jurt/com/sun/star/lib/util/NativeLibraryLoader.java
===================================================================
--- libreoffice-4.1.4~rc2.orig/jurt/com/sun/star/lib/util/NativeLibraryLoader.java	2013-12-16 14:35:06.314935257 +0100
+++ libreoffice-4.1.4~rc2/jurt/com/sun/star/lib/util/NativeLibraryLoader.java	2013-12-16 14:35:06.306935258 +0100
@@ -102,6 +102,8 @@
                 }
             }
         }
+        if (name.equals("soffice"))
+            return new File("/usr/lib/libreoffice/program/soffice");
         return null;
     }
 
